composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "crisu83/yii-auth",
+    "name": "boox/ws-fork/crisu83/yii-auth",
```

Comment:
Looks correct to me, but just in case lets make it clear that it is forked from `crisu83/yii-auth` and we do not claim that it is made by us.
You can add this to description: "Fork of crisu83/yii-auth"